README.md

```diff
@@ -62,6 +62,8 @@ public class SessionBackedChallengeNonceStore : IChallengeNonceStore
         }
         return null;
     }
+
+    public ChallengeNonce GetAndRemove() => GetAndRemoveImpl();
```

Comment:
This is misleading and should not be merged, the reasons are outlined in the [this comment](https://github.com/web-eid/web-eid-authtoken-validation-dotnet/pull/22#issuecomment-1665907818).